lib/sycamore/sycamore/transforms/split_elements.py

```diff
@@ -34,12 +34,17 @@ def __init__(self, child: Node, tokenizer: Tokenizer, maximum: int, **kwargs):
     def split_doc(parent: Document, tokenizer: Tokenizer, max: int) -> Document:
         result = []
         for elem in parent.elements:
+            # Ensure the _header does not take up more than a third of the tokens
+            # Also avoid max resursive depth error
+            if elem.get("_header") and len(tokenizer.tokenize(elem["_header"])) / max > 0.33:
+                del elem["_header"]
```

Comment:
Do we want to add a warning log message if the header is dropped?

Reply:
Do you want to replace _header with "Over token limit" in this case or some unique str? would it be useful printing a warning without knowing which element it's for?

Reply:
I'd rather have it log a warning. I don't want users to see "Over token limit" in their final product and not know what it is. The output is still usable even if it is missing the `_header`. You can add the `_header` text to the warning message.